lightly_studio_view/Makefile

```
@@ -71,3 +71,16 @@ else
 	nvm use; \
 	npm run test:unit -- --run'
 endif
+
+# Full-suite line coverage for the Fast Track coverage guardrail, which reads
+# coverage/coverage-final.json rather than running tests itself.
+.PHONY: test-coverage
+test-coverage:
+ifeq ($(OS),Windows_NT)
+	npm run test:unit -- --run --coverage --coverage.reporter=json
```

checkmake (0.3.2)

[warning] Target "npm run test" should be declared PHONY.

(phonydeclared)